arcgis-ios-sdk-samples/Layers/Manage sublayers/ManageSublayers.storyboard

```diff
@@ -29,9 +29,9 @@
                                 </constraints>
                                 <items>
                                     <barButtonItem style="plain" systemItem="flexibleSpace" id="J9K-Vc-wO5"/>
-                                    <barButtonItem title="Sublayers" id="CmR-Ri-JIK">
+                                    <barButtonItem title="Manage Sublayers" id="CmR-Ri-JIK">
                                         <connections>
-                                            <segue destination="cg1-iZ-LA9" kind="presentation" identifier="MapImageSublayersSegue" modalPresentationStyle="formSheet" id="MXS-1S-AVQ"/>
+                                            <segue destination="cg1-iZ-LA9" kind="presentation" identifier="MapImageSublayersSegue" id="MXS-1S-AVQ"/>
```

Comment:
Please keep the modal presentation style as `formSheet`. That way the view is not full screen in a regular width environment (such as on an iPad).

Reply:
Ah I was wondering what the difference was.